src/lib/notificationWorker.js

```diff
@@ -31,25 +43,10 @@ class NotificationWorker {
     const affectedAccounts = _([transfer.debits, transfer.credits])
       .flatten().pluck('account').map((account) => this.uri.make('account', account)).value()
     affectedAccounts.push('*')
-
-    let subscriptions = yield this.Subscription.findAll({
-      where: {
-        $and: [{
-          $or: [{
-            event: 'transfer.update'
-          }, {
-            event: 'transfer.*'
-          }, {
-            event: '*'
-          }]
-        }, {
-          subject: {
-            $in: affectedAccounts
-          }
-        }]
-      },
-      transaction
-    })
+    let subscriptions = yield transaction.from('subscriptions')
```

Comment:
I tested that transactions are working manually and had Nao review the method that I was using to test.